tests/base.py

```diff
@@ -38,6 +40,12 @@ def get_app(self):
                 MOCK_COURSE2: [MOCK_CLIENT_TOKEN1, MOCK_CLIENT_TOKEN2],
             },
         )
+
+        tornado.ioloop.PeriodicCallback(
+            lambda: callbacks.worker_heartbeat_callback(self.app.settings),
+            test_config.HEARTBEAT_INTERVAL * 1000,
```

Comment:
this is great! now we can test node failures! can you figure out a way to mock the `HEARTBEAT INTERVAL` in the config? If that is possible we will be able to test many features which we couldn't before due to this limitation!

Reply:
the reason is so that tests do not take so long to run. it can be a pain.

Reply:
I guess we could reduce HEARTBEAT_INTERVAL in the test_config? It's a separate config anyway

Reply:
Oh okay. Could it be reduced to like 2 secs?

Reply:
ok. fixed